packages/components/__tests__/DateField.spec.tsx

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, it } from 'vitest';
+
+import { render, screen } from '../../../test/utils';
+import { DateField, DateInput, DateSegment, Label, TimeField } from '../src';
```

Comment:
I think I missed the `DateSegment` PR, my bad.

We should add `font-variant-numeric: tabular-nums;` to the segments so that their width doesn't shift ever-so-slightly as the numbers change.

Reply:
Design wasn't a fan of the kerning with tabular-nums. Perhaps something worth following up on to see if we can achieve it with reduced letter spacing.